code/game/machinery/vending.dm

```diff
@@ -1320,3 +1316,128 @@
 	component_parts += new /obj/item/weapon/vending_refill/crittercare(0)
 	component_parts += new /obj/item/weapon/vending_refill/crittercare(0)
 	component_parts += new /obj/item/weapon/vending_refill/crittercare(0)
+
+//CODING BOOGALOO : FLIPPING OUT EDITION//

```

Comment:
This isn't really what comments should be for. If something is a WIP it shouldn't be merged. But up to you.